docs/mocksupport.rst

```diff
@@ -878,3 +878,58 @@ DMTF CIM schema download support
       :attributes:
 
    .. rubric:: Details
+
+
+.. _`Mock InMemory CIM Repository`:
+
+Mock in-memory CIM repository
+-----------------------------
+
+pywbem_mock provides a standard interface to a CIM repository that
+stores CIM classes, CIM instances, and CIM qualifier declarations.
+
+This repository implements an repository in memmory that is created and
+destroyed each time the FakedWBEMConnection class is constructed and
+destroyed.
+
+
+.. automodule:: pywbem_mock._inmemoryrepository
+
+.. autoclass:: pywbem_mock.InMemoryRepository
+   :members:
+
+   .. rubric:: Methods
+
+   .. autoautosummary:: pywbem_mock.InMemoryRepository
+      :methods:
+      :nosignatures:
+
+   .. rubric:: Attributes
+
+   .. autoautosummary:: pywbem_mock.InMemoryRepository
+      :attributes:
+
+   .. rubric:: Details
+
+.. _`Mock InMemory Object store`:
+
+Mock InMemory object store
```

Comment:
Change "InMemory" to "in-memory", to avoid partial symbol names and for consistency with the heading for the repository.

Reply:
Actually to be consistent it should be without the -. Changed to without dash and corrected caps on section headers.

Reply:
Is "inmemory" a valid term? When I google it, I only find "in-memory" (apart from a company named "InMemory").

Reply:
They all now read inmemory ...